filter_plugins/oo_filters.py

```diff
@@ -18,9 +18,9 @@
 from distutils.util import strtobool
 from distutils.version import LooseVersion
 from operator import itemgetter
-from ansible.module_utils.six.moves.urllib.parse import urlparse
 from ansible.parsing.yaml.dumper import AnsibleDumper
 from six import string_types
+from six.moves.urllib.parse import urlparse
```

Comment:
Ansible requires python-six already, but if we did want to use the vendored six for control hosts, we should use ansible.compat.six, rather than ansible.module_utils.six